deepchem/models/torch_models/dtnn.py

```diff
@@ -145,3 +148,134 @@ def forward(self, inputs: List[torch.Tensor]):
         dtnn_gather = nn.Dropout(self.dropout)(dtnn_gather)
         output = self.linear(dtnn_gather)
         return output
+
+
+class DTNNModel(TorchModel):
+    """Implements DTNN models for regression.
+
+    This class implements the Deep Tensor Neural Network (DTNN) [1]_.
```

Comment:
Can you add 1-2 paragraphs of explanation? Explain the core idea of the model and how it works

Reply:
Done